src/components/TodoContainer.js

```diff
@@ -1,77 +1,75 @@
-import React from "react"
-import TodoList from "./TodoList";
-import Header from "./Header";
-import InputTodo from "./InputTodo";
-import { v4 as uuidv4 } from "uuid";
+/* eslint-disable */
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import TodoList from './TodoList';
+import Header from './Header';
+import InputTodo from './InputTodo';
 import '../app.css';
 
 class TodoContainer extends React.Component {
   state = {
     todos: [
       {
         id: uuidv4(),
-        title: "Setup development environment",
-        completed: true
+        title: 'Setup development environment',
+        completed: true,
       },
       {
         id: uuidv4(),
-        title: "Develop website and add content",
-        completed: false
+        title: 'Develop website and add content',
+        completed: false,
       },
       {
         id: uuidv4(),
-        title: "Deploy to live server",
-        completed: false
-      }
-    ]
-   };
```

Comment:
Avoid using the index as key: In the todos array, id is used as the key. It's recommended to use a unique value as the key instead of using the index of the array, as it can cause issues with React's reconciliation algorithm when items are added or removed from the array. Kindly fix it.